tempesta_fw/cfg.c

```diff
@@ -888,6 +888,73 @@ tfw_cfg_map_enum(const TfwCfgEnum mappings[],
 EXPORT_SYMBOL(tfw_cfg_map_enum);
 
 /**
+ * Get value of attribute with name @attr_key.
+ * Return @default_val if the attribute is not found in the entry @e.
+ */
+const char *
+tfw_cfg_get_attr(const TfwCfgEntry *e, const char *attr_key,
+		 const char *default_val)
+{
+	const char *key, *val;
+	size_t i;
```

Comment:
This is not as pronounced in the code as the above, but still, it's everywhere as well. Shorter and simpler lines with variables should come first, and longer lines should come last. Unless it's necessary to do otherwise. The reasoning is the same, it's easier to read and comprehend that way. I would put it this way:
```c
size_t i;
const char *key, *val;
```